pkg/services/provisioning/plugins/plugin_provisioner_test.go

```diff
@@ -58,32 +40,10 @@ func TestPluginProvisioner(t *testing.T) {
 			},
 		}
 		reader := &testConfigReader{result: cfg}
-		ap := PluginProvisioner{log: log.New("test"), cfgProvider: reader}
+		ap := PluginProvisioner{log: log.New("test"), cfgProvider: reader, pluginSettings: &mockPluginsSettingsService{pluginSetting: &models.PluginSetting{}}}
 
 		err := ap.applyChanges(context.Background(), "")
 		require.NoError(t, err)
-		require.Len(t, sentCommands, 4)
-
-		testCases := []struct {
```

Comment:
Feels like we still want to keep some of these asserts to make sure arguments are provided as expected. Any reason removing this?